__tests__/cmds/login.test.ts

```diff
@@ -36,7 +36,7 @@ describe('rdme login', () => {
   it('should post to /login on the API', async () => {
     prompts.inject([email, password, project]);
 
-    const mock = getAPIMock().post('/api/v1/login').reply(200, { apiKey });
+    const mock = getAPIMock().post('/api/v1/login', { email, password, project }).reply(200, { apiKey });
```

Comment:
I really wish there were a way to force nock to error out if you send a payload to a mock that doesn't have one. I wonder if we have any other cases of this around here.